addons/TBMod_main/functions/fn_safe.sqf

```diff
@@ -57,7 +57,10 @@ else
         // publicVariable "TB_jip_safe";
     // };
     
-    if (!isNil "TB_safeInfo") then {terminate TB_safeInfo};
+    if (!isNil "TB_safeInfo") then {
```

Comment:
ABER.... `then {`
![Image of Yaktocat](https://media1.tenor.com/images/ee850bd1a6c4471f9b3a0a17919c18e4/tenor.gif?itemid=5834464)